<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings_DIalog_Serial_Port_Frame</class>
 <widget class="QFrame" name="Settings_DIalog_Serial_Port_Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QPushButton" name="pushButton_Rescan">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-scan ports</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_DMM">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Digial Multimeter (DMM) configuration</string>
   </property>
   <widget class="QLabel" name="label_DMM_port">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DMM Port:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DMM_port">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>23</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_DMM_baudrate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Baudrate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DMM_databits">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DataBits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DMM_stopbits">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>StopBits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DMM_parity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DMM_baudrate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>53</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DMM_databits">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>83</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DMM_stopbits">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>113</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DMM_parity">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>143</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_DMM_flowcontrol">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FlowControl:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DMM_flowcontrol">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>173</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_BC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>421</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Battery Controller (BC) configuration</string>
   </property>
   <widget class="QLabel" name="label_BC_port">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OLED Port:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_BC_port">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>23</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_BC_databits">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DataBits:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_BC_baudrate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>53</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_BC_stopbits">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>113</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_BC__parity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_BC_stopbits">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>StopBits:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_BC_databits">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>83</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_BC_parity">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>143</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_BC_baudrate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Baudrate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_BC_flowcontrol">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FlowControl:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_BC_flowcontrol">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>173</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_DMM_Test_Connection">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Test DMM </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_BC_Test_Connection">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Test BC</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
